### DISCUSSION TITLE
Does nvim-treesitter parse `fish` code?

### DISCUSSION THREAD

**Discussion by yuujay** [nvim-treesitter/nvim-treesitter#2322] [Q&A] [answered]
I have the treesitter installed for fish on my nvim 0.6.1 on my mac. But, when I view fish files from nvim I do not see syntax being highlighted. Is there any configuration that needs to be added?

`
<img width="886" alt="Screen Shot 2022-01-21 at 2 06 25 PM" src="https://user-images.githubusercontent.com/3604600/150588728-7afea2c2-8c38-441e-9b33-06dec8d6551d.png">
<img width="318" alt="Screen Shot 2022-01-21 at 2 07 46 PM" src="https://user-images.githubusercontent.com/3604600/150588729-e82bed64-8356-49d7-8c15-c2ce6d342976.png">
`

**Comment by stsewd** [comment, marked as answer]
Did you installed the fish parser? `:TSInstall fish` or does your buffer have the correct filetype? `:set ft?`

**Reply by yuujay** [reply]
I had it installed with `:TSInstall fish`, I figured that the parsing is off by default and I have to enable that setting.

```lua
require('nvim-treesitter.configs').setup({
  highlights = {
     enable = true,
  }
})
```

**Reply by adrian5** [reply]
You can mark this as **Answered** then.
